src/Okta.Sdk.IntegrationTest/UserScenarios.cs

```diff
@@ -303,16 +303,20 @@ public async Task UpdateUserProfile(string nickName)
             {
                 // Update profile
                 createdUser.Profile.NickName = nickName;
+                createdUser.Profile.AdditionalProperties = new Dictionary<string, object>();
+                createdUser.Profile.AdditionalProperties["homeworld"] = "Planet Earth";
```

Comment:
This requires that the property `homeworld` is first added to the user profile schema in an Org via the console. This means that this IT is bound to fail if run in a new Org that does not have this item added.

Reply:
Ok, I think in Java SDK it is not a good idea to do this as it is run in PDVs across multiple Orgs and will likely throw failures.

Reply:
Yeah, that makes sense.